src/main/scala/mesosphere/marathon/api/v2/PodsResource.scala

```diff
@@ -31,4 +50,207 @@ class PodsResource @Inject() (
   def capability(@Context req: HttpServletRequest): Response = authenticated(req) { _ =>
     ok()
   }
+
+  // TODO(jdef) force parameter? probably since this supports deployment; but what does it actually
+  // mean to "force" a create?
+  @POST @Timed
+  def create(
+    body: Array[Byte],
+    @DefaultValue("false")@QueryParam("force") force: Boolean,
+    @Context req: HttpServletRequest): Response = authenticated(req) { implicit identity =>
+
+    withValid(unmarshalJson[PodDef](decodeBytes(body, req))) { podDef =>
```

Comment:
Hmm. I think we should always validate the internal model, not the json model. its also where normalization should live.